domains/thedev.json

```diff
@@ -5,6 +5,6 @@
     "email": "thedev132@gmail.com"
   },
   "record": {
-    "A": "152.70.127.223"
+    "A": ["152.70.127.223"]
```

Comment:
404?

Reply:
its an Nginx web server. I confirm its the IP of my server.

Reply:
@thedev132 What port is your website hosted on?

Reply:
You can access my website via its IPv6 address. http://[2603:c024:c008:1bff:133:7a14:f85e:2e28]/fullscreen.html

Reply:
It is hosted on port 80 but non accessible from IPv4